File: 2014-10-CSharp-Fundamentals-1/3. Operators Expressions and Statements/LogicalOperators/LogicalOperators.cs
using System;

class LogicalOperators
{
	static void Main()
	{
        bool a = true;
        bool b = false;
        Console.WriteLine(a && b); // False
        Console.WriteLine(a || b); // True
        Console.WriteLine(a ^ b); // True
        Console.WriteLine(!b); // True
        Console.WriteLine(b || true); // True
        Console.WriteLine(b && true); // False
        Console.WriteLine(a || true); // True
        Console.WriteLine(a && true); // True
        Console.WriteLine(!a); // False
        Console.WriteLine((5 > 7) ^ (a == b)); // False
	}
}

File: 2015-10-Java-Fundamentals-1/03. Operators-and-expressions/demos/LogicalOperators.java
public class LogicalOperators {
    public static void main(String[] args) {
        boolean a = true;
        boolean b = false;
        writeLine(a && b); // False
        writeLine(a || b); // True
        writeLine(a ^ b); // True
        writeLine(!b); // True
        writeLine(b || true); // True
        writeLine(b && true); // False
        writeLine(a || true); // True
        writeLine(a && true); // True
        writeLine(!a); // False
        writeLine((5 > 7) ^ (a == b)); // False
    }

    public static void writeLine(Object obj) {
        System.out.println(obj);
    }
}